src/harness/reference_models/dpa/dpa_mgr.py

```diff
@@ -432,6 +448,39 @@ def CheckInterference(self, sas_uut_active_grants, margin_db,
 
     return max_diff_interf <= margin_db
 
+  def __PrintStatistics(self, results, margin_db, dpa_name, channel):
+    timestamp = datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+    filename='%s DPA=%s channel=%s.csv' % (timestamp, self.name, channel)
+    logging.info('Saving stats for DPA %s, channel %s to file: %s', dpa_name, channel, filename)
+    with open(filename, 'w') as f:
+      # CSV header.
+      f.write('Latitude,Longitude,Azimuth (degrees),A_DPA (dBm),A_DPA_ref (dBm),A_DPA - A_DPA_ref,A_DPA + 144 dBm\n')
```

Comment:
May be we could replace 144 by the value from the kml if available (self.threshold?)

Reply:
This is what ITS suggested so I would prefer to keep it as is. Perhaps you can ask Joel from ITS if he would prefer your approach?

Reply:
It is true it could create confusion if a test is done on one of the DPA with a different threshold, then they will see big negative number which leads to the belief the test is badly failed.
The title could be named: "A_DPA+Prot_thresh" and use self.threshold in the addition below

Reply:
Joel concurs, so I've changed the filename to be (e.g.) `2018-10-22 07_11_12 DPA=BATH channel=(3620, 3630) threshold=-144.0.csv` and the format fo the file to be

```
Latitude,Longitude,Azimuth (degrees),A_DPA (dBm),A_DPA_ref (dBm),A_DPA - A_DPA_ref,A_DPA - threshold
43.9055555600,-69.8138888900,0.0000000000,-179.7476485399,-144.0000000000,-35.7476485399,-35.7476485399
43.9055555600,-69.8138888900,1.5000000000,-179.7476485399,-144.0000000000,-35.7476485399,-35.7476485399
...
```